### PR TITLE
Update UI for comments

### PR DESCRIPTION
**Pull request by SMahil**
Update UI for Comments

#903

### CODE REVIEW

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 1]
```diff
@@ -1,37 +1,55 @@
-<div *ngIf="workItem" class="col-md-12 col-sm-12 col-xs-12" id="wi-comment">
-    <div class="comment-content" id="wi-comment-content">
+<div *ngIf="workItem" class="col-md-12 col-sm-12 col-xs-12 no-padding" id="wi-comment">
+    <div class="comment-content " id="wi-comment-content">
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: Extra space added after comment-content class name.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss [hunk 2]
```diff
@@ -2,63 +2,70 @@
...
+      padding-left: em(36);
+      h5 {
+          margin-top: em(0);
+          margin-bottom: em(5);
+      }
+      .createTimeBlock {
+        line-height: 1;
+      }
+      p {
+        margin: 0
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss]: semicolon missing.

**Comment by debloper** [comment]
I can see the placeholder text for the create comment input being broken. Can you confirm?

**Comment by SMahil** [comment]
@debloper Can you share the screenshot

**Comment by naina-verma** [comment]
[test]

**Comment by ldimaggi** [comment]
Is that the issue?

```

        },
        'creator': {
          'data': {
            'id': 'widct-user2',
            'type': 'identities'
          }
        },
        'comments': {
          'links': {
            'self': '',
            'related': ''
          }

```

Where do you see the broken text?  (In this context, what does "broken' mean? Thx!)

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss [hunk 3]
```diff
@@ -2,65 +2,86 @@
...
+      /* The min-height is same as form-control's default height */
         height: auto;
-        min-height: 26px;
-
+        min-height: em(26);
+        margin-bottom: 0;
         /* Helper class, should be moved out when almEditable is extended to support placeholders */
         &.placeholder {
-            color: $color-pf-black-400;
+            color: $color-pf-black-200;
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss]: This color is very light and hardly visible with white background. Can we use the same placeholder color which we are using for description field?

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 4]
```diff
@@ -1,37 +1,57 @@
...
+                              </h5>
+                               <p class="col-md-12 col-xs-12 col-sm-12 no-padding" id="{{'comment_body_' + counter}}">{{comment.attributes.body}}</p>
+                          </div>
+                      </div>
+                      </div>
                 </div>
-                <p class="col-md-12 col-xs-12 col-sm-12" id="{{'comment_body_' + counter}}">{{comment.attributes.body}}</p>
-            </div>     
+                <div class="panel-footer">
+                  <span>View more comments</span>
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: This should be cursor pointer.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss [hunk 5]
```diff
@@ -2,65 +2,86 @@
...
             }
         }
     }
 }
+@media (max-width: $screen-sm-min) {
+  .comment-content {
+    .comments-wrap {
+      > div {
+        .comment-body {
+          .createTimeBlock{
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss]: Need a space before {.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss [hunk 6]
```diff
@@ -2,65 +2,86 @@
...
+                  width: 7em;
+                  line-height: 1.5;
+                }
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss]: Please indent this block properly.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.spec.ts [hunk 7]
```diff
@@ -10,7 +10,7 @@ import {
 
 import { Location } from '@angular/common';
 import { SpyLocation } from '@angular/common/testing';
-import { DebugElement } from '@angular/core';
+import { DebugElement, Component } from '@angular/core';
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.spec.ts]: Why you import Component here? I can't see any use of this in this file.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss [hunk 8]
```diff
@@ -2,65 +2,86 @@
...
+    border-left: none;
+    border-right: none;
+    .panel-heading{
+      border: none;
     }
-    #wi-comment-add-comment {
-        /* Overrides pf's form-control style's height property */
-        /* The min-height is needed to avoid zero-height element */
-        /* The min-height is same as form-control's default height */
+    .wi-comment-input{
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss]: Please give a space before {.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.spec.ts (hunk 7 above)
Comment (SMahil) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.spec.ts]: Somehow it got added while debugging the Unit tests. Will Remove it

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 9]
```diff
@@ -1,37 +1,57 @@
...
-                            </strong>
-                        </h5>
-                    </div>
+          <div class="panel panel-default wi-comment-panel" >
+              <div class="panel-heading">
+                <h3 class="panel-title" (click)="isCollapsedComments = !isCollapsedComments">
+                  <i class="fa fa-icon"
+                    [ngClass]="{'fa-angle-right': isCollapsedComments,
+                        'fa-angle-down': !isCollapsedComments}"></i>
+                  Comments
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: Need to display the comments count. Can display a static number for now.

Comment (SMahil) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: Should we add it dynamic content directly in another PR?

Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: Sure @SMahil that would be great.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 10]
```diff
@@ -1,37 +1,57 @@
...
+                          </div>
+                      </div>
+                      </div>
                 </div>
-                <p class="col-md-12 col-xs-12 col-sm-12" id="{{'comment_body_' + counter}}">{{comment.attributes.body}}</p>
-            </div>     
+                <div class="panel-footer">
+                  <span>View more comments</span>
+                  <p class="pull-right">
+                    <span>5 </span> of <span>20</span>
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: This needs to be made dynamic. Will this be done in this PR?

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html (hunk 4 above)
Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: View more behaviour needs to be implemented. I think the UX for this is in prpgress.

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html (hunk 10 above)
Comment (SMahil) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: UX is already there but I am keeping this PR only for HTML CSS change. The more functional changes will do it in separate PR only

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html (hunk 4 above)
Comment (SMahil) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: UX is already there but I am keeping this PR only for HTML CSS change. The more functional changes will do it in separate PR only

Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: Ok

File: src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html (hunk 10 above)
Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html]: Ok

**Comment by SMahil** [comment]
[test]
